static/app/views/performance/transactionSummary/transactionTags/tagsHeatMap.tsx

```diff
@@ -164,7 +167,7 @@ const TagsHeatMap = (
       <TransitionChart loading={loading} reloading={reloading}>
         <TransparentLoadingMask visible={reloading} />
 
-        <HeatMapChart series={series} {...chartOptions} />
+        <HeatMapChart visualMaps={visualMaps} series={series} {...chartOptions} />
```

Comment:
splitting `visualMaps` from `chartOptions` isn't necessary here right? or did typescript yell at you for it?

Reply:
I wanted to call it out as explicit since it's basically required for styling the heatmap.